src/App.vue

```diff
@@ -1,5 +1,26 @@
 <template>
 	<div id="app" class="flex flex-col h-full">
 		<router-view class="flex-auto" />
+		<Alert />
+		<Backdrop>
```

Comment:
What is backdrop and why we have spinner and modal component inside backdrop? Shouldn't we have also alert inside backdrop ?

Reply:
Backdrop is a full-screen wrapper with slightly darker, yet still transparent background color. It creates a contrast between the upper layers of the app such as modals and the background - usually the page itself. 

The reason why Modal and Spinner are in the Backdrop is that those comopnents are then rendered in `<slot></slot>` of the `Backdrop.vue` component.

The Alert component needs no backdrop, therefore it's not in that component.